release-notes/20221103-privileged-scc.md

```diff
@@ -0,0 +1,4 @@
+<!-- markdownlint-disable MD041 -->
+The privileged SCC permission for Submariner components in OCP is set now by
+creating a separate ClusterRole and ClusterRoleBinding resources instead of
+manipulating the system privileged SCC CR.
```

Comment:
```suggestion
The privileged SCC permission for Submariner components in OCP is set now by
creating separate `ClusterRole` and `ClusterRoleBinding` resources instead of
manipulating the system privileged SCC resource.
```